src/components/navbar/index.tsx

```diff
@@ -74,15 +85,32 @@ const UserContainer = styled.div`
   margin-right: 16px;
 `;
 
+const UserMenu = styled(Menu)`
+  min-width: 100px;
+`;
+const UserDropdown = styled(Dropdown)`
+  min-height: 50px;
+`;
+const UserAvatar = styled(Avatar)`
+  margin-right: 10px;
+`;
+
 interface NavBarProps {
   readonly tokens: UserAuthenticationReducerState['tokens'];
+  readonly contacts: ContactsReducerState['contacts'];
 }
 
-const NavBar: React.FC<NavBarProps> = ({ tokens }) => {
+const NavBar: React.FC<NavBarProps> = ({ tokens, contacts }) => {
   const history = useHistory();
   const location = useLocation();
   const dispatch = useDispatch();
 
+  useEffect(() => {
+    if (asyncRequestIsComplete(tokens)) {
+      dispatch(getContactInfo());
```

Comment:
Might have a conflict here cause I am also doing the same thing here with `getRequestStatuses` in my ticket, but it'll be as simple as including both of them